src/thunderbird_accounts/authentication/signals.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+from django.db.models.signals import post_delete, pre_save
+from django.dispatch.dispatcher import receiver
+
+from thunderbird_accounts.authentication.models import User
+from thunderbird_accounts.authentication.utils import delete_cache_allow_list_entry
+
+
+@receiver(post_delete, sender=User)
+def user_post_delete(sender, instance, **kwargs):
```

Comment:
I don't see `sender` being used

Reply:
Looks like it's working without sender. So I'll pop that off.